### PR TITLE
wcm 0.20.0 (new formula)

### PR DESCRIPTION
**Pull request by corporateshark**
Added the WCM Commander file manager.

### CODE REVIEW

**Comment by MindTooth** [comment]
http://wcm.linderdaum.com/wcm-commander-0-20-0-released/

File: wcm.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  depends_on "cmake" => :build
+  depends_on "freetype"
+  depends_on "libssh2"
+  depends_on :x11
+  depends_on :macos => :mavericks
+
+  needs :cxx11
+
+  def install
```
Comment (DomT4) [inline wcm.rb]: Should be `ENV.cxx11` under `def install` as well here.

File: wcm.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    system "cmake", ".", "-DWITH_SMBCLIENT=OFF", *std_cmake_args
+    system "make", "all", "-j#{ENV.make_jobs}", "-B"
```
Comment (DomT4) [inline wcm.rb]: Specifying `-j` shouldn't be necessary.

File: wcm.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+class Wcm < Formula
+  homepage "http://wcm.linderdaum.com"
```
Comment (DomT4) [inline wcm.rb]: Please add a `desc` above the homepage here. Should be less than 80 characters.

File: wcm.rb (hunk 1 above)
Comment (corporateshark) [inline wcm.rb]: Added

**Comment by DomT4** [comment]
Thanks for the quick response! Could you possibly squash your commits into one? Thanks!

**Comment by corporateshark** [comment]
Done.

**Comment by DomT4** [comment]
Thanks, that seems to have made our test-bot a little happier. Why won't it build on Mountain Lion and below, out of interest?

**Comment by corporateshark** [comment]
I experienced multiple problems with x11/clang configurations of your test-bot. Just disabled those OS X versions to work around those issues.

**Comment by DomT4** [comment]
They may have been `cxx11` failures. I wonder if it'd work without `depends_on :macos => :mavericks` now we've added the `cxx11` ENV.

**Comment by corporateshark** [comment]
I have no idea.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-x11/commit/658bef8993a47e4b6cc4687062ba7bb53d66f59e. Thanks for your contribution to Homebrew/x11, We appreciate it! :smiley_cat:

**Pull request closed without merging**